samples/algorithms/HiddenShift.qs

```diff
@@ -34,14 +34,11 @@ namespace Sample {
                 nQubits
             );
             let hiddenShift = ResultArrayAsInt(hiddenShiftBitString);
-            Fact(
```

Comment:
I think we should add a comment to indicate what is expected.

Reply:
I did.